obelix/panoramix/operator/consumer.go

```diff
@@ -67,17 +67,17 @@ func New(ctx context.Context) (*Operator, error) {
 // It returns the next task's namespace, associated method name and parameters.
 // e.g.: GOOGLE_CREATE_NEW_EVENT becomes google CreateNewEvent with its associated parameters.
 func (o *Operator) getNextTask(initialActionID string) (string, string, []interface{}, error) {
-	client, err := protos.NewClient("9000")
+	client, err := task.NewClient("9000")
```

Comment:
`9000` should be takes from environment

Reply:
@PtitLuca that your part of code

Reply:
@TomChv I know, it will be fixed in a new PR, no need to create a linked issue.